adoc/admin-security-certificates.adoc

```diff
@@ -318,25 +325,31 @@ chmod 600 my-cluster/pki/front-proxy-ca.key
 
 After this process, bootstrap the cluster with `skuba node bootstrap`.
 
-== Replace Server Certificate signed by a Trusted CA Certificate
+=== Replace Server Certificate signed by a Trusted CA Certificate
 
 {productname} uses `oidc-dex` and `oidc-gangway` servers to do authentication and authorization.
 Administrators might choose to replace these server's certificates by issuing a trusted CA certificate after cluster deployment.
 This way, the user does not have to add specific certificates to their trusted keychain.
 
+* With The Trusted CA Key
++
+Please ref to <<addon_certificates_rotation>> on how to use cert-manager and reloader to issue `oidc-dex` and `oidc-gangway` certificate signed by trusted CA certificate/key.
+
+* Without The Trusted CA Key
++
 [WARNING]
 ====
-The custom trusted CA certificate key is not handled by skuba.
+Due to the custom trusted CA certificate key is not in {kube} cluster.
```

Comment:
```suggestion
Because the custom trusted CA certificate is not in {kube} cluster, administrators must handle server certificate rotation manually before the certificate expires.
```